### DISCUSSION TITLE
Welcome to Marlin-2-for-Sapphire-Plus-V2-gen2 Discussions!

### DISCUSSION THREAD

**Discussion by khunpoum** [khunpoum/Marlin-2-for-Sapphire-Plus-V2-gen2#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Slapdash16** [comment]
Thank you so much for your work on this. It's the only solution I've been able to get working on my factory sapphire plus.

**Comment by Kn9z** [comment]
Hello everyone. I stitched my sapphire plus, for some reason the screen is turned upside down, and the z limit switch does not work either. Can anyone help with this? I have version 1.1. Z-axis stepper motors are controlled by different drivers but there is a sync belt. Z-axis limit switch only one.
